ros.el

```diff
@@ -598,6 +598,21 @@
   (let ((real-flags (seq-filter  (lambda (flag) (not (string= flag "ISOLATED")) )flags)))
     (ros-compile-action (ros-dump-colcon-action :workspace ros-current-workspace :verb "build" :flags real-flags :post-cmd (when test (concat "colcon test --packages-select " package " && colcon test-result --verbose"))))))
 
+(defun ros-find-debug-executable ()
+  "Search for, then launch a ROS2 node in GDB mode."
+  (interactive)
+  (if (eq (ros-current-version) 1) (error "Debug does not currently support ros1")
+  (let* ((lib-path (ros-shell-command-to-string "env | grep LD_LIBRARY_PATH"))
```

Comment:
maybe rather do ``echo $LD_LIBRARY_PATH``?

Reply:
An echo would be more elegant, however the output gets cut by the sed in 'ros-shell-command-to-string (I don't understand why)  and I figured the env | grep was less intrusive than modifying the expression, or creating a duplicate shell command function.